terraform/aws/components/nhais/variables.tf

```diff
@@ -151,6 +151,13 @@ variable "nhais_mesh_host" {
   default = ""
 }
 
+variable "NHAIS_MESH_CERT_VALIDATION" {
+  type = string
+  description = "false will disable certificate validation for SSL connection"
+  #default = "false"
+  default = "false"
```

Comment:
has to be changed to 'true' after uploading sub ca cert

Reply:
NHAIS_MESH_CERT_VALIDATION needs to go into locals_env_variables.tf - this is where non-sensitive environment variables are defined

This file contains variable that are internal to terraform. You should also create an "nhais_mesh_cert_validation" variable here. The environment variable will get its "valueFrom" this variable.

Then in terraform/aws/etc you can set a default value for each environment.

build1: true (this *should* use OpenTest but we don't actually use build1 for anything)
vp: false (this uses fake-mesh(
ptl: true (this uses PTL/INT MESH API - it is important that correct CA certs are configured in the secret since we're testing this currently)